### DISCUSSION TITLE
Add template/preset feature with tags etc. for new documents

### DISCUSSION THREAD

**Discussion by pkoerner81929** [LeoKlaus/Paperparrot#130] [Feature Requests]
When scanning or importing new docs, allow selecting an import template with predefined tags, correspondent etc. That would save a lot of manual, repeating steps after importing. Comparable to Consume-folder structure at paperless-ngx

e.g.

1. Template "Receipts, Taxi expenses"
TAGS predefined "2023", "Tax relevant", "Taxi", "Munich"
TYPE: "Invoice"

**Comment by dast5022** [comment]
Also very helpful would be to pre-set the permissions of that uploaded files.
